### ISSUE TITLE
Contrast on draft badge is insufficient

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2391]
Currently 1.83

![image](https://user-images.githubusercontent.com/255909/66520528-2c459200-ea9e-11e9-8a36-3806c47da756.png)
